src/cmd1.cc

```diff
@@ -652,34 +567,6 @@ void carry(int pickup)
 }
 
 
-/*
- * Handle player hitting a real trap
- */
-static void hit_trap(void)
-{
-	bool_ ident = FALSE;
-
-	cave_type *c_ptr;
-
-
-	/* Disturb the player */
-	disturb(0);
-
-	/* Get the cave grid */
-	c_ptr = &cave[p_ptr->py][p_ptr->px];
-	if (c_ptr->t_idx != 0)
-	{
-		ident = player_activate_trap_type(p_ptr->py, p_ptr->px, NULL, -1);
```

Comment:
Isn't there some `.cc` file where player_activate_trap_type could be removed from? (I can't find it in this diff.)